src/modules/middleSection.js

```diff
@@ -7,14 +8,68 @@ const fetchAll = async () => {
     arr = response;
   });
 };
+
+const closePopupModal = () => {
+  const closeIcon = document.querySelector('.close-icon');
+  closeIcon.addEventListener('click', (e) => {
+    e.preventDefault();
+    if (!commentModal.classList.contains('hide-modal')) {
+      commentModal.classList.add('hide-modal');
+    }
+  });
+};
+
+const showCommentModal = () => {
+  const commentBtns = document.querySelectorAll('.comment-btn');
+  commentBtns.forEach((commentBtn) => {
+    commentBtn.addEventListener('click', () => {
+      const btnId = commentBtn.getAttribute('id');
+      const movieDetails = arr[btnId];
+
+      const modal = `
+        <div class="header-container">
+          <div class="img-container">
+            <img src="${movieDetails.image.original}" class="mov-img" alt="Movie Image">
+          </div>
```

Comment:
I would suggest to resize the image to make it fit the container.

Reply:
Noted thanks, will do as suggested.